lib/matplotlib/tests/test_mathtext.py

```diff
@@ -364,3 +364,58 @@ def test_mathtext_to_png(tmpdir):
     mt = mathtext.MathTextParser('bitmap')
     mt.to_png(str(tmpdir.join('example.png')), '$x^2$')
     mt.to_png(io.BytesIO(), '$x^2$')
+
+
+def test_mathtext_fontset():
```

Comment:
A few seconds?  Thats far too long.  

I think we need to add an image (just one please) to the repo for this.  Then just use our normal figure comparison machinery.  If you are going to just compare if two files are equal you can just save to bytesIO instead of to disk.  But in this case I think we need the file comparison.